static/app/components/events/interfaces/frame/stacktraceLink.tsx

```diff
@@ -230,6 +231,34 @@ export function StacktraceLink({frame, event, line, disableSetup}: StacktraceLin
     refetch();
   };
 
+  const filePath =
+    frame.filename && frame.lineNo ? frame.filename + ':' + frame.lineNo : '';
+
+  const {onClick: handleCopyPath} = useCopyToClipboard({
+    text: filePath,
+    successMessage: t('File path copied to clipboard'),
+    errorMessage: t('Failed to copy file path'),
+    onCopy: () => {
+      trackAnalytics('integrations.stacktrace_file_path_copied', {
+        view: 'stacktrace_issue_details',
+        organization,
+        group_id: event.groupID ? parseInt(event.groupID, 10) : -1,
+        ...getAnalyticsDataForEvent(event),
+      });
+    },
+  });
+
+  const copyButtonProps = {
```

Comment:
It might help to enforce that these are `ButtonProps`? 

```diff
+ import type { ButtonProps } from 'sentry/components/core/button';
```
```suggestion
  const copyButtonProps: ButtonProps = {
```